### PR TITLE
Different encoding section language

### PR DESCRIPTION
**Pull request by skef**
After going through #174 a few times it wasn't really working for me. A lot of what was needed was there but the ordering didn't make sense. Topics seemed like they came up and then went past without leading into one another. 

Especially given that this section is non-normative, I feel it could be more helpful; the sort of thing where you read it from start to finish once or twice and have a good idea of the broad strokes of what an encoder has to do. So I tried restructuring the info that was there and adding some more, which resulted in what you see here.

This was too large to count as PR feedback and too intrusive and complex to just stage as a PR-PR like #172. So I'm just putting this up separately. If you like this then we can work on it further "here", or you can grab what you like and add it and we can work on it over in #174.

***
<a href="https://pr-preview.s3.amazonaws.com/skef/IFT/pull/178.html" title="Last updated on May 25, 2024, 2:09 AM UTC (a8118ca)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/IFT/178/ed3d73b...skef:a8118ca.html" title="Last updated on May 25, 2024, 2:09 AM UTC (a8118ca)">Diff</a>

### CODE REVIEW

File: Overview.bs [hunk 1]
```diff
@@ -1956,30 +1960,127 @@ functionality/data from all of its patch files even if those were included in th
...
+types leave the possibility of less typical patterns open. For example, one could use [[#per-table-brotli]] patches for all
+content other than the glyph tables but then use another set of [[#per-table-brotli]] patches for those tables rather than
+[[#glyph-keyed]] patches, and each of these could in theory be partially invalidating—leaving them mutually dependent but 
+independent of one another.
+
+The two patch types are similar in that they are at least Partially Invaliding, so that the choice of one from a table (IFT or IFTX)
+means that all others in that table are invalidated.  Assuming the patched result does not represent the fully expanded font,
+that patch will typically alter the IFT or IFTX table it was was listed in to add a new set of patches to further extend the
+font. This means that the total set of [[#brotli]] or [[#per-table-brotli]] patches forms a graph, in which each font subset
+in the segmentation is a node and each patch is a graph.  This also means that patches of these types are typically downloaded
```
Comment (garretrieger) [inline Overview.bs:1997]: should be "each patch is an edge"

File: Overview.bs [hunk 2]
```diff
@@ -1956,30 +1960,127 @@ functionality/data from all of its patch files even if those were included in th
...
 ------------------------------------------------
 
 <em>This section is not normative.</em>
 
 The details of the encoding process may differ by encoder and are beyond the scope of this document. However, this section provides
-guidance that encoder implementations may want to consider.
+guidance that encoder implementations may want to consider, and that can be important to reproducing the appearance and behavior of
+an existing font file when producing an [=incremental font|incremental=] version of that font.
+
+<b>About [[#brotli]] and [[#per-table-brotli]] patches</b>
```
Comment (garretrieger) [inline Overview.bs:1972]: Would be good to link back to the section on invalidation (4.1) somewhere in this sub-section.

Comment (garretrieger) [reply Overview.bs:1972]: You could also link to the specific invalidation type definitions whenever you reference them (eg. [=Partial Invalidation|Partially Invalidating=])

File: Overview.bs [hunk 3]
```diff
@@ -2010,57 +2111,71 @@ encode_node(full_font, base_font, cur_def, subset_definitions):
...
+    of those in the [=glyph closure=] of the [=font subset definition=].
 
-Brotli based patches can modify the mapping table in the current font subset, which allows a patch to update the mapping table
-to point to a different set of patches which are valid against the result of the patch application. This allows a graph to be formed
-where a font subset is the node and each patch is an edge.
+Assuming the subsetter does its job accurately, the glyph closure requirement is a consequence of the requirement for equivalent
+behavior: Suppose there is a [=font subset definition=] such that the subsetter includes glyph *i* in its subset, but an encoder that
+produces [[#glyph-keyed]] patches omits glyph *i* from the set of patches that correspond to that definition. If the subsetter is
+right, that glyph must be present to retain equivalent behavior when rendering some combination of the codepoints and features in 
+the definition. And this means that the [=incremental font=] will not behave equivalently when rendering that combination.
```
Comment (garretrieger) [inline Overview.bs:2139]: Drop the "and" or merge these two sentences.

**Review by garretrieger — APPROVED** [review]
Thanks, this alternate version looks good to me. Just a few minor comments.

**Comment by skef** [comment]
I addressed the review comments. Should @svgeesus look at this before merging?

**Review by svgeesus — APPROVED** [review]
Yes, this is much clearer and provides good guidance.

**Comment by garretrieger** [comment]
@skef Can you rebase this and then I'll merge.

**Comment by skef** [comment]
Should be rebased now

**Pull request merged**